app/models/agents/weather_agent.rb

```diff
@@ -135,22 +117,12 @@ def validate_location
     end
 
     def validate_options
-      errors.add(:base, "service must be set to 'darksky' or 'wunderground'") unless wunderground? || dark_sky?
+      errors.add(:base, "The Weather Underground API has been disables since Jan 1st 2018, please switch to DarkSky") if wunderground?
```

Comment:
There's a trivial typo `disables` and maybe the date should be updated.

Reply:
Good cach ;-). I just pushed an update